code/modules/surgery/organs/augments_arms.dm

```diff
@@ -243,8 +242,10 @@
 /obj/item/organ/cyberimp/arm/flash/Initialize()
 	. = ..()
 	if(locate(/obj/item/assembly/flash/armimplant) in items_list)
-		var/obj/item/assembly/flash/armimplant/F = locate(/obj/item/assembly/flash/armimplant) in items_list
+		var/obj/item/assembly/flash/armimplant/F = new(src)
 		F.I = src
+		items_list += F
+		return TRUE
```

Comment:
No, I only told you to modify the riot shield emag_act , not the `Initialize` procedure of every `cyberimp/arm` that start out with a flash. Undo this change.